phoenix-core/src/main/antlr3/PhoenixSQL.g

```diff
@@ -496,6 +498,11 @@ show_node returns [ShowStatement ret]
     |   SHOW SCHEMAS (LIKE pattern=string_literal)? { $ret = factory.showSchemasStatement(pattern); }
     ;
 
+// Parse a describe statement. SHOW CREATE STATEMENT tablename ...
+show_create_statement_node returns [ShowCreateStatement ret]
+    :   SHOW CREATE STATEMENT tablename=from_table_name { $ret = factory.showCreateStatement(tablename); }
```

Comment:
Can we make it "SHOW CREATE TABLE" than "SHOW CREATE STATEMENT" to adhere to other databases like mysql.

https://dev.mysql.com/doc/refman/8.0/en/show-create-table.html
https://docs.databricks.com/sql/language-manual/sql-ref-syntax-aux-show-create-table.html
https://prestodb.io/docs/current/sql/show-create-table.html

Reply:
Would be better to support to get the statements for index as well as views also. May be another JIRA.

Reply:
This from_table_name can be actually a view or index table as well.
https://github.com/apache/phoenix/blob/9b76bd51c86faea117d963b8fc033dbfef647d47/phoenix-core/src/main/antlr3/PhoenixSQL.g#L1129

It is already working for views and for indexes as well. I added some tests to be sure.  Istvan was in the favour of naming this SHOW CREATE STATEMENT as it works for views and indexes as well.  

In case we chose to rename it to SHOW CREATE TABLE, should it work for views and indexes as well or Do you think those should be separate commands?

Reply:
bq.In case we chose to rename it to SHOW CREATE TABLE, should it work for views and indexes as well or Do you think those should be separate commands?
Same should work for table,index and views as well instead of separate commands.